README.md

```diff
@@ -202,9 +203,16 @@ list. These are community maintained templates and may not be updated often.
 </p>
 </details>
 
+### Adding Support for a New Application
+
+To add your own template, submit a pull request to this repository to add your
+repository to the "Unofficial Templates" list below in the relevant category.
+**Recommended repository naming scheme: [style-name]-[template-name]** (with dashes as
+separators).
+
 ## Schemes
 
-To add your own scheme, submit a pull request to the [base16-schemes repo](https://github.com/base16-project/base16-schemes) and add your scheme.
+To add a Base16 scheme, simply submit a pull request to the [base16-schemes repo](https://github.com/base16-project/base16-schemes) and add your scheme.
```

Comment:
Perhaps remove "simply" here too?

Reply:
Looking more for "big picture "thoguhts - make sure I'm headed the right direction, I'm still editing quite a lot...

Reply:
right, ignore this then :)